lightning/src/util/config.rs

```diff
@@ -100,6 +100,13 @@ pub struct ChannelHandshakeLimits {
 	///
 	/// Default value: 0.
 	pub min_funding_satoshis: u64,
+	/// Maximum allowed satoshis when a channel is funded, this is supplied by the sender and so
+	/// only applies to inbound channels.
+	///
+	/// Default value: 2^24.
+	//  Note that this default was chosen because it was the maximum funding amount prior to wumbo
+	//  channels being introduced.
+	pub max_funding_satoshis: u64,
```

Comment:
Why have a config parameter? If we advertise support for wumbo, wouldn't it be surprising to a peer if they couldn't open a channel >= 2^24?

Reply:
Users are likely to implement it anyway via the `accept_channel` hooks, plus its a really important security parameter - what level of risk are you willing to tolerate if the channel funds somehow get lost because you went offline too long?